.dockerignore

```diff
@@ -1,6 +1,11 @@
+.git
 *.pyc
 .idea
-.git
+.coverage
 .tox
+.cache
+dist/
+build/
 docs/
```

Comment:
True, I thought of it mostly as a packaged executable, so I wouldn't expect people to go inside the container and build the docs. Do you think that may happen?